genesis/utils/gltf.py

```diff
@@ -0,0 +1,395 @@
+from io import BytesIO
+from urllib import request
+import numpy as np
+import pygltflib
+import trimesh
+from scipy.spatial.transform import Rotation as R
+from PIL import Image
+
+import genesis as gs
+from . import mesh as mu
+
+ctype_to_numpy = {
+    5120: np.int8,  # BYTE
+    5121: np.uint8,  # UNSIGNED_BYTE
+    5122: np.int16,  # SHORT
+    5123: np.uint16,  # UNSIGNED_SHORT
+    5124: np.int32,  # INT
+    5125: np.uint32,  # UNSIGNED_INT
+    5126: np.float32,  # FLOAT
+}
+
+type_to_count = {
+    "SCALAR": (1, []),
```

Comment:
What about just doing `math.prod` when the total number of elements is required. I don't like redundant information it makes access to both information more convoluted (ie `.[0]`, `.[1]` or unpacking)